src/config_file.c

```diff
@@ -106,6 +106,12 @@ gboolean load_config(const gchar *filename, RaucConfig **config, GError **error)
 			goto free;
 		}
 		g_key_file_remove_key(key_file, "system", "efi-use-bootnext", NULL);
+	} else if (g_strcmp0(c->system_bootloader, "custom") == 0) {
+		c->custom_script = key_file_consume_string(key_file, "system", "bootloader-custom-script", NULL);
+		if (!c->custom_script) {
+			g_printf("No custom script defined. defaulting to /usr/bin/foobar as default");
+			c->custom_script = g_strdup("/usr/bin/foobar");
+		}
```

Comment:
Why not just a hard error?

Reply:
Changed to cause an error if not defined. Thanks